### DISCUSSION TITLE
Clarification about Seed Shapes

### DISCUSSION THREAD

**Discussion by YasirKusay** [bbuchfink/diamond#658] [General]
Hi, 

I have a few questions regarding shapes. For each shape, do you use the same seed set (except varying the positions within each seed required for a match). What are the differences between the number shapes and weights per each sensitivity option for the most recent diamond version (v2.11.0)? According to your original paper, the default settings uses 4 shapes of length 15-24 and a weight of length 12.

Finally, aside from number of shapes and weights, are there any other differences between the sensitivity options?

**Comment by bbuchfink** [comment]
The seeds are based on the same sequences for each shape. You can find the shape configurations here: https://github.com/bbuchfink/diamond/blob/master/src/search/setup.cpp
There are many internal cutoff values that also depend on the sensitivity mode.
